### DISCUSSION TITLE
Welcome to OpenDHT Discussions!

### DISCUSSION THREAD

**Discussion by aberaud** [savoirfairelinux/opendht#643] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by n7tae** [comment]
Hello. I'm building a network for ham radio that uses OpenDHT so that nodes on the network can publish their configuration. Server nodes publish their possibly changing configuration in a dht::Value keyed by the reflector's callsign. Then, client nodes then just need to know the reflector's callsign to figure out how to join and link to a server.
